.github/workflows/ci.yml

```diff
@@ -1,180 +1,228 @@
 name: CI
 
-## Only run when:
-##   - manually triggered
-##   - PR's are (re)opened
-##   - push to master (i.e. merge develop -> master)
-
 on:
   push:
     branches:
       - master
-  pull_request:
+      - development
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
   workflow_dispatch:
     inputs:
       tag:
         description: "The tag to create (optional)"
         required: false
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
+    ## might be better to use inclusive v exclusive paths here, ex:
+    # paths:
+    #   - "**.rs"
+    #   - "**.clar"
+  pull_request_review:
+    types:
+      - submitted
+    paths-ignore:
```

Comment:
I don't think this is a valid member of `pull_request_review`

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review

it's in the docs and works as expected where the workflow triggers when a review is submitted (conditionals prevent running unless it's an approved request).